pandas/tests/io/test_parquet.py

```diff
@@ -931,6 +931,17 @@ def test_read_parquet_manager(self, pa, using_array_manager):
         else:
             assert isinstance(result._mgr, pd.core.internals.BlockManager)
 
+    def test_read_empty_array(self, pa):
+        df = pd.DataFrame(
```

Comment:
since you are at it, can you add regular types as well, e.g. int, float, object strings, datetime w/tz, categorical (i think we have another test which constructs these all), so can make that a fixture and just send it thru empty.

Reply:
hmm we speciifically wanted to test Int64 right? try adding some type from https://github.com/pandas-dev/pandas/blob/master/pandas/conftest.py#L832

Reply:
Thank you for your comments.
I added `Int64` and `UInt8`.

Reply:
For the nullable types, I think we should also check `use_nullable_dtypes=True` (since that was the original issue).  think you can pass this through to `check_round_trip` using `read_kwargs`

Reply:
@mzeitlin11 
Thank you for your comments.
When I use `use_nullable_dtypes`, I found some error in test `int64` or `bool`.
Do you mean that I should not use `int64` or `bool`?
```
E           AssertionError: Attributes of DataFrame.iloc[:, 0] (column name="value") are different
E           
E           Attribute "dtype" are different
E           [left]:  int64
E           [right]: Int64

pandas/tests/io/test_parquet.py:211: AssertionError

E           AssertionError: Attributes of DataFrame.iloc[:, 0] (column name="value") are different
E           
E           Attribute "dtype" are different
E           [left]:  bool
E           [right]: boolean

pandas/tests/io/test_parquet.py:211: AssertionError
```

Reply:
`use_nullable_types` should only round trip for nullable types (so `Int64` and `UInt8` in your parameterization).

Reply:
@mzeitlin11 
Do I have to test other regular types?
@jreback  says that I should add regular types like float and object strings.

Reply:
Yeah the more testing the better! Easiest solution would probably just to test `use_nullable_dtypes` in a separate test with only nullable types like `Int64` and `UInt85

Reply:
can you add boolean here as well; separately we DO want to test empyt for all types